### PR TITLE
Add Internal Server Errors + PastelWonderland

### PR DESCRIPTION
**Pull request by Stereo528**
Add the Infamous Internal Server Errors people might get when refreshing the site. Both a few funny (I hope) ones always referencing it, and adding it as a bad thing.
Also added PastelWonderland, A mod me and one other has worked on, which is still partly in dev but has a GitHub release.

### CODE REVIEW

File: data/author [hunk 1]
```diff
@@ -152,3 +152,5 @@ Portable Jukebox:Will BL
 Lapis Warps:Will BL
 GVault:Will BL
 Gshops:Will BL
+PastelWonderland:Stereo528
+PastelWonderland:HiItsDevin_
```
Comment (Siuolplex) [inline data/author:161]: I think you should add Devin to Consistency+ Devs as well. Hes considered a developer in the discord.

**Review by Siuolplex — CHANGES_REQUESTED** [review]

File: data/author (hunk 1 above)
Comment (ghost) [reply data/author:161]: oh uh.. you dont have to if you dont want to, i barely contribute...

Comment (IMS212) [reply data/author:161]: do it anyway

Comment (piperpaperrose) [reply data/author:161]: Done, anyone who has dev role is dev smh.

**Review by Siuolplex — APPROVED** [review]

**Comment by frqnny** [comment]
I’m concerned over the amount of scenarios might debalance others

**Comment by Stereo528** [comment]
I can remove a few of the more "duplicate" ones.

**Pull request merged**